.github/ISSUE_TEMPLATE/issue-report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Issue report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
Is this supposed to be different from a bug report? Perhaps this template should be a "Feature Request" instead.